templates/zerver/help/code-blocks.md

```diff
@@ -127,13 +127,14 @@ be happy to help you out.
 
 ## Related articles
 
-[Math blocks][math-block], [spoiler blocks][spoiler-block], and [quote
-blocks][quote-block] use similar fenced block syntax.
+[Math blocks][math-block], [Kroki diagram blocks][kroki-diagram-block],
+[spoiler blocks][spoiler-block], and [quote blocks][quote-block] use similar fenced block syntax.
```

Comment:
Let's put the `Korki diagram blocks` option at the end of this list, since it's the most obscure.